press/press/doctype/release_group/test_release_group.py

```diff
@@ -239,3 +240,94 @@ def test_cant_set_min_greater_than_max_workers(self):
 	def test_update_available_shows_for_first_deploy(self):
 		rg = create_test_release_group([create_test_app()])
 		self.assertEqual(deploy_information(rg.name).get("update_available"), True)
+
+	def test_fetch_environment_variables(self):
+		rg = create_test_release_group([create_test_app()])
+		environment_variables = [
+			{"key": "test_key", "value": "test_value", "internal": False},
+			{"key": "test_key_2", "value": "test_value", "internal": False},
+			{"key": "secret_key", "value": "test_value", "internal": True},
+		]
+		for env in environment_variables:
+			rg.append("environment_variables", env)
+		rg.save()
+		rg.reload()
+
+		fetched_environment_variable_list = get_list(
+			"Release Group Variable",
+			fields=["name", "key", "value"],
+			filters={"parenttype": "Release Group", "parent": rg.name},
+		)
+		self.assertEqual(len(fetched_environment_variable_list), 2)
+		internal_environment_variables_keys = [
+			env["key"] for env in environment_variables if env["internal"]
+		]
+		non_internal_environment_variables_keys = [
+			env["key"] for env in environment_variables if not env["internal"]
+		]
+		for env in fetched_environment_variable_list:
+			self.assertNotIn(env.key, internal_environment_variables_keys)
+			self.assertIn(env.key, non_internal_environment_variables_keys)
```

Comment:
`get_list` returning 0 records in CI, but when testing locally working correctly .

Reply:
Fixed !
It was due to changee of team in `setUp` method